### PR TITLE
Accommodate incident report manifest ingestion

### PR DESCRIPTION
**Pull request by jccraft-uw**
This commit updates the manifest cron job to use new required parameters and
adds a new job to ingest the incident report manifest.

The file lives on Google Drive, so a new directory is added for the
necessary environment variable.

These changes depend on:
seattleflu/specimen-manifests#15
seattleflu/id3c#160

### CODE REVIEW

File: crontabs/id3c-production [hunk 1]
```diff
@@ -37,10 +37,13 @@ XDG_RUNTIME_DIR=/home/ubuntu/run
...
 
-# Manifest is uploaded to AWS at arbitrary times now, so check for new records every 10m.
+# The aliquot manifest is uploaded to AWS at arbitrary times now, so check for new records every 10m.
 # This is often the blocker for the presence/absence ETl, so run more frequently than
 # the presence/absence ETL to avoid extended delay of results.
-*/10 * * * * ubuntu chronic fatigue envdir $ENVD/hutch/ /opt/specimen-manifests/update-unattended --push-and-upload
+*/10 * * * * ubuntu chronic fatigue CONFIG="/opt/specimen-manifests/season-2-AWS-S3.yaml" MANIFEST="/opt/specimen-manifests/season-2-manifest.ndjson" DIFF_FILE="aliquot-diff.ndjson" envdir $ENVD/hutch/ /opt/specimen-manifests/update-unattended --push-and-upload
+
+# Process the incident report manifest from Google Sheets every 30m.
+*/30 * * * * ubuntu chronic fatigue CONFIG="/opt/specimen-manifests/incident-report-manifest.yaml" MANIFEST="/opt/specimen-manifests/incident-report-manifest.ndjson" DIFF_FILE="incident-report-diff.ndjson" envdir $ENVD/google/ /opt/specimen-manifests/update-unattended --push-and-upload
```
Comment (kairstenfay) [inline crontabs/id3c-production:46]: The timing of this job (every 30 minutes) works well given that we run the [SCAN RoR cron job](https://github.com/seattleflu/backoffice/blob/master/crontabs/scan-return-of-results) at 5 and 35 past every hour.

**Comment by kairstenfay** [comment]
Worth mentioning that this depends on https://github.com/seattleflu/specimen-manifests/pull/15 and https://github.com/seattleflu/id3c/pull/160

**Comment by jccraft-uw** [comment]
[Ingest the lab's incident report spreadsheet to get data for bad samples](https://trello.com/c/McH4xpP9/695-ingest-the-labs-incident-report-spreadsheet-to-get-data-for-bad-samples)

File: crontabs/id3c-production (hunk 1 above)
Comment (tsibley) [inline crontabs/id3c-production:46]: We could avoid the long string of environment vars by putting them into envdirs.

Comment (kairstenfay) [reply crontabs/id3c-production:46]: That could be nice!

Comment (jccraft-uw) [reply crontabs/id3c-production]: That does make things nice! I've made that change.

**Review by joverlee521 — APPROVED** [review]

**Pull request merged**